store/src/java-test/com/zimbra/cs/service/ModifyPrefsTest.java

```diff
@@ -36,12 +36,10 @@
 import com.zimbra.common.util.L10nUtil;
 import com.zimbra.cs.account.Account;
 import com.zimbra.cs.account.Provisioning;
-import com.zimbra.cs.mailbox.MailItem;
```

Comment:
these changes have already been done in ZCS2948.

Reply:
Reverted